internal/docker/docker.go

```diff
@@ -56,11 +56,13 @@ type Client struct {
 	logger               *zap.Logger
 }
 
-func NewDockerClient(config *Config, logger *zap.Logger) (*Client, error) {
+func NewDockerClient(config *Config, logger *zap.Logger, opts ...docker.Opt) (*Client, error) {
```

Comment:
If we have reverted the `go-connections` version, does this still need to happen?
Or is this just a safe nice to have?

Reply:
Yeah, it doesn't need to be there anymore, but imo it doesn't hurt to leave it in, and it could be nice to have for future purposes or similar issues.